src/main/java/com/it4045/pokemonviewer/enterprise/dto/Pokedex.java

```diff
@@ -8,5 +8,5 @@
 class Pokedex {
 
     @SerializedName("pokemon")
-    private List<Pokemon> pokemon;
+    private List<Pokemon> pokemons;
```

Comment:
As the tag uses "List" here. Then, its is better to use plural instead of singular form.

Reply:
Yes, agreed.